platform-service-framework/src/test/java/org/hypertrace/core/serviceframework/PlatformServiceTest.java

```diff
@@ -92,11 +92,11 @@ private void startService(final PlatformService service) {
 
   @Test
   public void testMetricInitialization() {
-    PlatformService service = getService(Map.of("service.name", "test-service",
+    PlatformService service = getService(Map.of("service.name", "sample-app",
```

Comment:
any reason for the changes in here? The default tag change should be transparent (And is, if I'm reading the test changes correctly)

Reply:
With new changes, these tests are failing when they are tested with different default tags.
These tests are simulating a reload (lifecycle stop and start) kind of scenario. However, for commonTags there is no provision for cleanup/destroy. We need to destroy the registry itself and recreate it.

Reply:
Isn't that a caught regression though? In the prior implementation, it reset its state on start/stop and it no longer does. Seems like it would be easy enough to reset the meter registry, right?